src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -2718,18 +2718,20 @@ self =>
         val vparamss = paramClauses(name, contextBoundBuf.toList, ofCaseClass = false)
         newLineOptWhenFollowedBy(LBRACE)
         var restype = fromWithinReturnType(typedOpt())
+        def msg(what: String) = s"procedure syntax is $what. convert procedure `$name` to method by adding `: Unit`."
+        def defmsg(what: String) = s"procedure syntax is $what. convert procedure `$name` to method by adding `: Unit =`."
```

Comment:
@SethTisue `=` I think is in the error message when it is a definition. Do you have a negative test link that shows otherwise?

Reply:
sorry, I missed that, nm.